tests/test_titiler_pgstac.py

```diff
@@ -159,5 +159,6 @@ def test_tiles(rio, app):
     assert meta["width"] == 256
     assert meta["height"] == 256
 
+    # tile is outside mosaic bbox, it should return 404 (NoAssetFoundError)
     response = app.get(f"/tiles/{search_bbox}/{z}/{x}/{y}?assets=cog")
     assert response.status_code == 404
```

Comment:
This test is failing now. The tile requested is outside the mosaic bbox but still return assets now

Reply:
pgstac v0.4.0 makes CQL2-json the default query language. All the tests are using the old query syntax which in pgstac get rewritten into CQL1. Adding "filter-lang":"cql-json" should allow these tests to still work until updated to CQL2-json.

Reply:
🙏 thanks @bitner it fixed it 😄